README.md

```diff
@@ -78,11 +78,11 @@ simply run:
 
 ```bash
 bash ops/build.sh
-``` 
+```
 
 which should output the detailed information about what are thrown into the
 Jar file while building the executable Jar file to `target/push-to-cloud-service.jar`.
 
 You could run it with `java -jar target/push-to-cloud-service.jar`, which will invoke
-the main function in the `start.clj` namespace, which is currently the only module with 
+the main function in the `start.clj` namespace, which is currently the only module with
 `:gen-class`.
```

Comment:
:sigh:

Reply:
Sorry, I set vscode to strip all trailing whitespace. Is this wrong?

Reply:
No, it's awesome.

Reply:
It's awesome! (I just did the same)